src/state.js

```diff
@@ -0,0 +1,12 @@
+const grid = [
+  0, 1, 1, 1, 1, 1, 1, 0,
```

Comment:
this is a good format for now, may consider changing it down the road, maybe make a Grid primitive?

Reply:
👍